unsloth/scripts/run-unsloth.sh

```
@@ -0,0 +1,735 @@
+  --tee <number>               TEE offer id to use for Unsloth
+  --config <file>              Path to general spctl config (default: ./config.json)
+  --use-configuration <file>   Use existing Unsloth configuration JSON and skip tunnels launcher creation
+  --suggest-only               Build prompts and show non-interactive command, but skip any spctl calls and file writes
+  -h, --help                   Show this help
+EOF
+}
+
+# Parse arguments (optional, keep minimal)
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    *=*)
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 112.

(SC2221)